bin/rspec

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
Ummm. Where did this file come from?

Reply:
To be consistent with the other binstubs (but agree it doesn't belong here)

Reply:
OK - I'm happy for it either to be left here, or pulled and put into another branch/PR